jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ContentProducer.java

```diff
@@ -89,11 +89,11 @@
     boolean hasContent();
```

Comment:
Now that this class isn't dealing with `Content`s but `Chunk`s this method should be renamed. Maybe to `hasChunk()`?